Formula/gap.rb

```diff
@@ -39,14 +40,19 @@ def install
 
     # Create a symlink `bin/gap` from the `gap` binary
     bin.install_symlink libexec/"gap" => "gap"
-
+    libexec.install_symlink pkgshare/"pkg" => "pkg"
+    
     ohai "Building included packages. Please be patient, it may take a while"
-    cd libexec/"pkg" do
+    cd pkgshare/"pkg" do
       # NOTE: This script will build most of the packages that require
       # compilation. It is known to produce a number of warnings and
       # error messages, possibly failing to build several packages.
-      system "../bin/BuildPackages.sh", "--with-gaproot=#{libexec}"
+      system "../../../libexec/bin/BuildPackages.sh", "--with-gaproot=#{libexec}"
     end
+
+    # Now remove the symlink to pkgshare/"pkg", and put one to /usr/local/share/gap/pkg, so we can access global packages too
+    rm libexec/"pkg"
+    ln_s "/usr/local/share/gap/pkg", libexec/"pkg"
```

Comment:
Do you really want to put things into /usr/local -- or into the brew prefix? Which isn't /usr/local anymore on e.g. M1 macs